spec.html

```diff
@@ -39990,9 +39990,13 @@ <h1>Promise.all ( _iterable_ )</h1>
         <emu-alg>
           1. Let _C_ be the *this* value.
           1. Let _promiseCapability_ be ? NewPromiseCapability(_C_).
+          1. Let _promiseResolve_ be GetPromiseResolve(_promiseCapability_).
+          1. If _promiseResolve_ is an abrupt completion, then
+            1. IfAbruptRejectPromise(_promiseResolve_, _promiseCapability_).
+            1. Return Completion(_promiseResolve_).
```

Comment:
```suggestion
            1. Return _promiseResolve_.
```
it's already an abrupt completion.

Reply:
This step actually don't make sense - [IfAbruptRejectPromise](https://tc39.es/ecma262/#sec-ifabruptrejectpromise) is a macro, and it causes the invoking function to return if passed an abrupt completion. So control can never reach here.

I think this step can be omitted entirely.

Reply:
That makes much more sense.

Reply:
Ah, I didn't realize it was macro! That does indeed make so much more sense...